tests/utils.py

```diff
@@ -263,10 +256,11 @@ def __init__(self, lightning_dir, bitcoin_dir, port=9735, random_hsm=False):
 
         # Last 32-bytes of final part of dir -> seed.
         seed = (bytes(re.search('([^/]+)/*$', lightning_dir).group(1), encoding='utf-8') + bytes(32))[:32]
+        if not random_hsm:
+            with open(os.path.join(lightning_dir, 'hsm_secret-regtest'), 'wb') as f:
```

Comment:
CI complains due to this, since *this* PR does not separate filenames by network type.

Reply:
Yes, overzealous cherry pick. Will fix.